config/configuration.json

```diff
@@ -1079,6 +1079,27 @@
     },
     "minHighwayType": "tertiary"
   },
+  "TallBuildingCheck": {
+    "enabled": true,
+    "bufferDistanceMeters": 1600.0,
+    "minDatasetSizeForStatsComputation": 50.0,
+    "maxLevelTagValue": 100.0,
+    "outlierMultiplier": 18.0,
+    "invalidHeightCharacters": ["~","`","!","@","#","$","%","^","&","*","(", ")","-","_","+","=","[","[","}","]","|","\\",":",";","<",",",">","?","/"],
+    "magicNumbers": {
```

Comment:
Since these are just instruction indices, I don't think we need to make these configurable values.

Reply:
addressed